api/en/292_OAuth2_Authorization_Endpoint.md

```diff
@@ -1,25 +1,22 @@
-# OAuth2.0 Authorization Endpoint(\_\_authz)
+# OAuth 2.0 Authorization Endpoint
 
 ## Overview
-OAuth2 Authorization Endpoint  
-This API is the OAuth2 authorization endpoint, for utilizing the Personium on JS application and native application.
+OAuth 2.0 Authorization Endpoint of a Cell. Usually it returns an HTML to render on Web Browsers, however depending on the state of Cookies and so on, it may directly redirect to the specified URL with grant code, access token or id token.
 
 ### Restrictions
-For scope=openid, only the following response_type can be specified.  
+For issuing ID tokens of OpenID Connect, only the following combination of parameters setting is supported.
 
-* response_type=id_token
-* response_type=code
+* scope=openid
+* response_type=id_token or code
 
 ## Request
 ### Request URL
 ```
-{CellName}/__authz
+{CellURL}/__authz
 ```
 
 ### Request Method
-GET : Authentication form request  
-POST : Authorization processing request (Request Token Authentication, Request Code Authentication, Request ID Token Authentication)

```

Comment:
Actually Unit Manager and HomeApp do use POST method to retrieve token. However, both should not be considered as normal Apps. Therefore, we should have a special page that explains how POST method is used.

Reply:
Thank you for pointing it out. I raised an issue for it.
https://github.com/personium/api-references/issues/179